diff --git a/printable.module b/printable.module
index e4b3da2..4f4b7e1 100755
--- a/printable.module
+++ b/printable.module
@@ -263,34 +263,6 @@ function printable_preparing_pdf() {
   );
 }

-/**
- * Make file urls absolute if we're generating a PDF.
- *
- * @param string $uri
- *   URI to be altered.
- */
-function printable_file_url_alter(&$uri) {
-  if (printable_preparing_pdf()) {
-    $scheme = \Drupal::service('file_system')->uriScheme($uri);
-
-    if ($scheme) {
-      $path = \Drupal::service('file_system')->realpath($scheme . '://');
-      $uri = ($path ? $path . '/' : '') . substr($uri, strlen($scheme) + 3);
-    }
-    else {
-      $base_url = \Drupal::request()->getSchemeAndHttpHost();
-
-      if (substr($uri, 0, strlen($base_url)) == $base_url) {
-        $uri = substr($uri, strlen($base_url) + 1);
-      }
-
-      $uri = DRUPAL_ROOT . '/' . $uri;
-    }
-
-    $uri = 'printable://' . $uri;
-  }
-}
-
 /**
  * Make Link URLs absolute if we're generating a PDF.
  *
