diff --git a/docroot/.htaccess b/docroot/.htaccess
index d7f40c216..0ff4487db 100644
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -135,6 +135,13 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
+  # Ensure non-live sites prevent robots from crawling.
+  RewriteCond %{HTTP_HOST} "^test\." [NC,OR]
+  RewriteCond %{HTTP_HOST} "^live\." [NC]
+  RewriteCond %{REQUEST_URI} /robots.txt [NC]
+  RewriteCond %{QUERY_STRING} "!^test$" [NC]
+  RewriteRule ^ no_robots.txt [L]
+
   # Redirect common PHP files to their new locations.
   RewriteCond %{REQUEST_URI} ^(.*)?/(install.php) [OR]
   RewriteCond %{REQUEST_URI} ^(.*)?/(rebuild.php)
